warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__technical_contact_listed.sql

```diff
@@ -7,35 +7,39 @@ feed_info_clean AS (
     SELECT * FROM {{ ref('feed_info_clean') }}
 ),
 
-feed_info_clean_check AS (
-    SELECT
-        calitp_itp_id,
-        calitp_url_number,
-        calitp_extracted_at,
-        calitp_deleted_at,
-        CASE
-            WHEN feed_contact_email IS NOT null THEN "PASS"
-        ELSE "FAIL"
-        END AS status
-    FROM feed_info_clean
-),
-
 -- Joins our feed-level check with feed_guideline_index, where each feed will have a row for every day it is active
-daily_feed_info_clean_check AS (
+daily_technical_contact_check AS (
     SELECT
         t1.date,
         t1.calitp_itp_id,
         t1.calitp_url_number,
         t1.calitp_agency_name,
         t1.check,
         t1.feature,
-        t2.status
+        CASE
+            WHEN t2.feed_contact_email IS NOT null THEN "PASS"
+            ELSE "FAIL"
+        END AS status,
+        COUNT(*)
     FROM feed_guideline_index AS t1
-    LEFT JOIN feed_info_clean_check AS t2
+    LEFT JOIN feed_info_clean AS t2
        ON t1.calitp_itp_id = t2.calitp_itp_id
       AND t1.calitp_url_number = t2.calitp_url_number
       AND t1.date >= t2.calitp_extracted_at
       AND t1.date < t2.calitp_deleted_at
+    GROUP BY 1,2,3,4,5,6,7
+),
+
+daily_technical_contact_check_dedupe AS (
```

Comment:
I am just wondering about calling this "dedupe" since it seems like a simple select statement -- is this intended to be a `select distinct`?

Reply:
You're right, the language is weird there. The deduping is done by the COUNT(*) in the previous subquery, and so the subquery titled "...dedupe" does nothing other than (arguably) neaten it up. i'm just going to remove it.